2021-09-30 12:29:13,389 INFO    MainThread:41734 [wandb_setup.py:_flush():69] setting env: {}
2021-09-30 12:29:13,389 INFO    MainThread:41734 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-30 12:29:13,389 INFO    MainThread:41734 [wandb_init.py:_log_setup():342] Logging user logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-2-data-exploration-and-preparation/exercises/exercise_6/starter/wandb/run-20210930_122913-1v8r0luf/logs/debug.log
2021-09-30 12:29:13,389 INFO    MainThread:41734 [wandb_init.py:_log_setup():343] Logging internal logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-2-data-exploration-and-preparation/exercises/exercise_6/starter/wandb/run-20210930_122913-1v8r0luf/logs/debug-internal.log
2021-09-30 12:29:13,390 INFO    MainThread:41734 [wandb_init.py:init():375] calling init triggers
2021-09-30 12:29:13,390 INFO    MainThread:41734 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-30 12:29:13,390 INFO    MainThread:41734 [wandb_init.py:init():424] starting backend
2021-09-30 12:29:13,390 INFO    MainThread:41734 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-09-30 12:29:13,402 INFO    MainThread:41734 [backend.py:ensure_launched():135] starting backend process...
2021-09-30 12:29:13,408 INFO    MainThread:41734 [backend.py:ensure_launched():139] started backend process with pid: 41752
2021-09-30 12:29:13,410 INFO    MainThread:41734 [wandb_init.py:init():429] backend started and connected
2021-09-30 12:29:13,412 INFO    MainThread:41734 [wandb_init.py:init():477] updated telemetry
2021-09-30 12:29:13,413 INFO    MainThread:41734 [wandb_init.py:init():500] communicating current version
2021-09-30 12:29:13,949 INFO    MainThread:41734 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-30 12:29:13,949 INFO    MainThread:41734 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-30 12:29:14,177 INFO    MainThread:41734 [wandb_init.py:init():540] starting run threads in backend
2021-09-30 12:29:14,673 INFO    MainThread:41734 [wandb_run.py:_console_start():1601] atexit reg
2021-09-30 12:29:14,673 INFO    MainThread:41734 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-30 12:29:14,673 INFO    MainThread:41734 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-30 12:29:14,674 INFO    MainThread:41734 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-30 12:29:14,674 INFO    MainThread:41734 [wandb_init.py:init():565] run started, returning control to user process
2021-09-30 12:29:14,853 INFO    MainThread:41734 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-30 12:29:14,853 INFO    MainThread:41734 [wandb_run.py:_restore():1543] restore
2021-09-30 12:29:17,222 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 1241
}

2021-09-30 12:29:19,318 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 1241
}

2021-09-30 12:29:19,528 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 1241
}

2021-09-30 12:29:20,228 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 5423
}

2021-09-30 12:29:20,335 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 7793
}

2021-09-30 12:29:20,437 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 7793
}

2021-09-30 12:29:20,541 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:20,648 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:20,757 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:20,864 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:20,970 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:21,327 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:21,435 INFO    MainThread:41734 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 12:29:22,520 INFO    MainThread:41734 [wandb_run.py:_show_files():1915] logging synced files
